MAUI/Shimmer/Customization.md

```diff
@@ -0,0 +1,239 @@
+
+## Fill
+
+The background color of the `SfShimmer` view can be customized by using the `Fill` property. The default value of `Fill` property is "#F3EDF7" Color.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<shimmer:SfShimmer x:Name="Shimmer" VerticalOptions="Fill" Type="Article" Fill="Pink">
+   <shimmer:SfShimmer.Content>
+      <StackLayout>
+         <Label 
+            Text="Content is loaded!!">
+         </Label>
+      </StackLayout>
+   </shimmer:SfShimmer.Content>
+</shimmer:SfShimmer>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+SfShimmer Shimmer = new SfShimmer()
+{
+      Type = ShimmerType.Article,
+      Fill = Colors.Pink,
+      Content = new Label
+      {
+         Text = "Content is loaded!"              
+      }
+};
+
+this.Content = Shimmer;
+
+{% endhighlight %}
+{% endtabs %}
+
+## WaveColor
+
+The color of wave in `SfShimmer` view can be customized by using the `WaveColor` property. The default value of `WaveColor` is `#F3EDF7` Color.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<shimmer:SfShimmer x:Name="Shimmer" VerticalOptions="Fill" Type="Article" WaveColor="#9A4EAE">
+   <shimmer:SfShimmer.Content>
+      <StackLayout>
+         <Label 
+            Text="Content is loaded!!">
+         </Label>
+      </StackLayout>
+   </shimmer:SfShimmer.Content>
+</shimmer:SfShimmer>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+SfShimmer Shimmer = new SfShimmer()
+{
+      Type = ShimmerType.Article,
+      WaveColor = Color.FromRgba("#9A4EAE"),
+      Content = new Label
+      {
+         Text = "Content is loaded!"              
+      }
+};
+
+this.Content = Shimmer;
+
+{% endhighlight %}
+{% endtabs %}
+
+## WaveWidth
+
+The width of wave in `SfShimmer` view can be customized by using the `WaveWidth` property. The default value of `WaveWidth` is `200`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<shimmer:SfShimmer x:Name="Shimmer" VerticalOptions="Fill" Type="Article" Fill="Pink" WaveColor="#9A4EAE"  WaveWidth="100">
+   <shimmer:SfShimmer.Content>
+      <StackLayout>
+         <Label 
+            Text="Content is loaded!!">
+         </Label>
+      </StackLayout>
+   </shimmer:SfShimmer.Content>
+</shimmer:SfShimmer>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+SfShimmer Shimmer = new SfShimmer()
+{
+      Type = ShimmerType.Article,
+      Fill = Colors.Pink,
+      WaveColor = Color.FromRgba("#9A4EAE"),
+      WaveWidth = 100,
+      Content = new Label
+      {
+         Text = "Content is loaded!"              
+      }
+};
+
+this.Content = Shimmer;
+
+{% endhighlight %}
+{% endtabs %}
+
+## WaveDirection
+
+The direction of wave in `SfShimmer` view can be customized by using the `WaveDirection` property. By default, wave direction is rendered by `Default` wave. The following wave directions are available in SfShimmer:
```

Comment:
The direction of the wave in the `SfShimmer` view can be customized by using the `WaveDirection` property. By default, wave direction is rendered by the `Default` wave. The following wave directions are available in SfShimmer:

Reply:
Content modified